.github/workflows/dotnetcore.yml

```
@@ -39,12 +36,17 @@ jobs:
       with:
         global-json-file: ./global.json
 
+    - name: Setup latest .NET 10 preview SDK
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: '10.0.x-preview'
```

```suggestion
        dotnet-version: '10.0.x'
        dotnet-quality: 'preview'
```

See: https://github.com/actions/setup-dotnet

---

In the past, we've had to specifically define the version of the preview

---

at the link Keegan included it says preview versions are included by default and it does seem to get installed... ¯\_(ツ)_/¯